src/store/root.store.ts

```diff
@@ -64,6 +64,14 @@ export default class RootStore {
     return location_text ? location_text : '';
   }
 
+  @computed
+  get newsFlashWidgetsMetaSegmentName(): string {
+    let {
```

Comment:
1. this should be const (fix line 63 if you can as well)
2. breaking destructing into 2 lines will make it more readable

Reply:
like this? : 
get newsFlashWidgetsMetaSegmentName(): string {
    const {
      location_info:
        { road_segment_name } } = this.newsFlashWidgetsMeta;
    return road_segment_name ? road_segment_name : '';
  }

Reply:
Actually - since you only need the inner property, you can do something better:
```
  get newsFlashWidgetsMetaSegmentName(): string {
    const { road_segment_name } =  this.newsFlashWidgetsMeta.road_segment_name;
    return ...;
  }

```